BGRSCRPT/iocBoot/iocBGRSCRPT-IOC-01/config.xml

```diff
@@ -4,6 +4,7 @@
   <ioc_desc>Background Script</ioc_desc>
   <ioc_details>IOC runs a background script from configurations area</ioc_details>
   <macros>
+    <macro name="SCRIPT_PATH" pattern="^.*$" description="Path to the Python background script." hasDefault="NO"/>
```

Comment:
Would be useful to give a simple example or details of the default.

This also helps people to know whether they need to type `c:\something`, `c:/something`, or `c:\\something`